flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/schema/SchemaOperator.java

```diff
@@ -127,4 +136,13 @@ RESPONSE sendRequestToCoordinator(REQUEST request) {
                     "Failed to send request to coordinator: " + request.toString(), e);
         }
     }
+
+    @Override
+    public void initializeState(StateInitializationContext context) throws Exception {
+        if (context.isRestored()) {
+            if (getRuntimeContext().getIndexOfThisSubtask() == 0) {
```

Comment:
I didn't catch up this limitation, could you explain this ?

Reply:
Because there may be multiple subtasks for the SchemaOperator, but we just want to clear the information in the SchemaRegistryHandler once during restart.

Reply:
Got it, it will better to add the explanation as code note